react_rails_getting_started.org

```diff
@@ -230,41 +218,39 @@ class HomeController < ApplicationController
     render_component message: 'World'
   end
 end
-
 #+END_SRC
 
-* The search path
+** The search path
 
   Where possible, you should stick with the conventions, but there are
-  times where you will not, for example, shared components.  The
+  times where you will not, for example, with shared components.  The
   search path for isomorphic components is described [[https://github.com/zetachang/react.rb#changing-the-top-level-component-name-and-search-path][here]] which says
 
 #+BEGIN_QUOTE
 Changing the top level component name and search path
 
-   You can control the top level component name and search path.
+You can control the top level component name and search path.
 
-   You can specify the component name explicitly in the
-   render_component method. render_component "Blatz will search the
-   for a component class named Blatz regardless of the controller
-   method.
+You can specify the component name explicitly in the
+render_component method. render_component "Blatz" will search the
+path for a component class named =Blatz= regardless of the controller
+method.
 
-   Searching for components normally works like this: Given a
-   controller named "Foo" then the component should be either in the
-   Components::Foo module, the Components module (no controller -
-   useful if you have just a couple of shared components) or just the
-   outer scope (i.e. Module) which is useful for small apps.
+Searching for components normally works like this: Given a
+controller named "Foo", then the component should be either in the
+=Components::Foo= module, the =Components= module (no controller -
+useful if you have just a couple of shared components) or just the
+outer scope (i.e. =Module=) which is useful for small apps.
 
-   Saying render_component "::Blatz" will only search the outer scope,
-   while "::Foo::Blatz" will look only in the module Foo for a class
-   named Blatz.
+Saying "render_component '::Blatz'" will only search the outer scope,
```

Comment:
I could not figure out a way to render the above expression is monospace font.  The quotes get in the way.  Hence, left it as quote.